core/templates/dev/head/dashboard/DashboardServices.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Standalone services for the creator dashboard page.
+ */
+
+oppia.constant('EXPLORATIONS_SORT_BY_KEYS', {
```

Comment:
I don't really think this dictionary is necessary, at least not here. For a function that is designed to sort a list of objects by some key, it makes sense to simply pass in the name of the key you are using to sort, rather than going through this dictionary first.

I know earlier you were using a list of the key names in order to check that the key passed in was valid. I think that's also unnecessary--the objects can check if they have that key with `hasOwnProperty`. Unless there are keys that the objects have which you don't want to allow?